Configuration/TypoScript/Main/setup.typoscript

```diff
@@ -206,7 +206,7 @@ plugin.tx_femanager {
           subject = TEXT
           subject.data = LLL:EXT:femanager/Resources/Private/Language/locallang.xlf:emailCreateUserConfirmationSubject
 
-          # Set CC receivers
+          # Set CC receivers (comma separatet list for multiple CC receivers)
```

Comment:
separatet => separated

For all changes below as well.

Reply:
Fixed